api/fake-seam-connect.ts

```diff
@@ -1,23 +1,26 @@
-import type { IncomingMessage, ServerResponse } from 'node:http'
+import { readFileSync } from 'node:fs'
+import path from 'node:path'
+import process from 'node:process'
 import { Readable, type Stream } from 'node:stream'
 
+import {
+  createFake as createFakeDevicedb,
+  type Fake as FakeDevicedb,
+} from '@seamapi/fake-devicedb'
 import { createFake } from '@seamapi/fake-seam-connect'
+import type { VercelRequest, VercelResponse } from '@vercel/node'
 import axios from 'axios'
 import getRawBody from 'raw-body'
 
 // eslint-disable-next-line import/no-relative-parent-imports
 import { seedFake } from '../.storybook/seed-fake.js'
 
-// Taken from seam-connect
-// Based on: https://stackoverflow.com/a/44091532/559475
-export const getRequestStreamFromBuffer = (requestBuffer: Buffer): Stream => {
-  const requestStream = new Readable()
-  // eslint-disable-next-line @typescript-eslint/no-empty-function
-  requestStream._read = () => {}
-  requestStream.push(requestBuffer)
-  requestStream.push(null)
-  return requestStream
-}
+// UPSTREAM: This line must have this precise format, otherwise
+// Vercel will not include the JSON file in the deployment.
+// https://github.com/vercel/next.js/discussions/14807#discussioncomment-146735
+const devicedbSeed = readFileSync(
+  path.join(process.cwd(), '.storybook', 'devicedb-seed.json')
+)
```

Comment:
Hmm i would just import the json directly

Reply:
I would need to enable `resolveJsonModule`, which normally would be ok, but for this project it will cause issues if this is used in `src` as [JSON modules](https://nodejs.org/api/esm.html#json-modules) are not usable yet.